### ISSUE TITLE
ORCID accounts are not getting email addresses

### ISSUE THREAD

**Issue by effigies** [OpenNeuroOrg/openneuro#3109]
### What went wrong?

Seen in admin user list:

![image](https://github.com/user-attachments/assets/0a921e59-861e-48e7-b603-80b3e1ad9758)

Doesn't seem to be universal, but these users cannot be given access to datasets.

### Expected behavior

Accounts should have emails.

### Desktop

- OS:
- Browser:
- Version:

### Phone

- Device:
- OS:
- Browser:
- Version:

**Comment by effigies** [comment]
Example dataset where ORCID login breaks assumption of emails as user ID: https://openneuro.org/datasets/ds005437/share

**Comment by effigies** [comment]
I have a FreshDesk issue where a user cannot have a dataset shared with them due to this bug: Freshdesk 2020.

**Comment by effigies** [comment]
I have a freshdesk issue where a user anticipates losing access to a gmail account. They would like to transition the accounts to their ORCID, but we don't have a way to do that if the ORCID doesn't have an associated email.

**Comment by nellh** [comment]
ORCID accounts do get an email if one was shared with OpenNeuro. We can't require it but maybe we should have a very obvious warning that this limits functionality?

I'd also like to allow sharing by another reference to the account but the unavoidable consequence is no notifications for users that didn't make an email accessible to OpenNeuro.

**Comment by effigies** [comment]
Can they re-authenticate to associate an email?

**Comment by nellh** [comment]
> Can they re-authenticate to associate an email?

Yes, I'll write some documentation for this today and see if I can come up with a visible place to put it if a user signed in with ORICD but has not yet made an email available.

**Comment by effigies** [comment]
> come up with a visible place to put it if a user signed in with ORICD but has not yet made an email available.

Maybe a yellow caution triangle ⚠️, something on the order of

![image](https://github.com/user-attachments/assets/a0342945-fe0e-4082-b83f-b9cb69c310c5)

with a link to the doc? (I assume we can get a slightly nicer style.)

**Comment by nellh** [comment]
Some updates from discussion today, planning a full page warning and disabling uploads until an email is added.
